Src/PerformanceCollector/Shared/Implementation/QuickPulse/Process/IQuickPulseProcessProvider.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.PerfCounterCollector.Implementation.QuickPulse
+{
+    using System;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Provider interface for Windows processes.
+    /// </summary>
+    internal interface IQuickPulseProcessProvider
+    {
+        void Initialize();
+
+        void Close();
+
+        IEnumerable<QuickPulseProcess> GetProcesses(out TimeSpan? totalTime);
```

Comment:
I've provided the doc summary. The name "totalTime" refers to the value of the _Total instance of the processor time counter, not sure how to make it much clearer.